dxr/templates/file.html

```diff
@@ -12,19 +12,24 @@
   <link href="{{ url_for('.static', filename='css/tree-selector.css') }}" rel="stylesheet" type="text/css" media="screen" />
 {% endblock %}
 
-{% block content %}
-  <div class="breadcrumbs">
-    {%- for path, name in paths_and_names -%}
-      {%- if not loop.first or loop.last -%}
-        <span class="path-separator">/</span><a href="{{ www_root }}{{ path }}">{{ name }}</a>
-      {%- else -%}
-        <a href="{{ www_root }}{{ path }}">{{ name }}</a>
-      {%- endif -%}
-    {%- endfor -%}
-  </div>
-
-  {{ tree_menu(tree_tuples, tree) }}
+{% block nav_bar %}
```

Comment:
It seems like we should be able to factor out some of this as well, which is repeated in folder.html, perhaps by moving much of it to the default `nav_bar` block in layout.html and then overriding it with blankness in the (one?) place it isn't needed, in error.html.